apps/qubit/modules/settings/actions/securityAction.class.php

```diff
@@ -62,11 +62,13 @@ protected function populateSecurityForm()
     $limitAdminIp = QubitSetting::getByName('limit_admin_ip');
     $requireSslAdmin = QubitSetting::getByName('require_ssl_admin');
     $requireStrongPasswords = QubitSetting::getByName('require_strong_passwords');
+    $googleMapsApiKey = QubitSetting::getByName('google_maps_api_key');
```

Comment:
I'm not sure if the security page is the best place for the Google Maps JavaScript API key. It's not even a secret, is it? I'd personally prefer to see it (by order of preference): 1) in the same place where you enable the map element (under default page elements, see: https://imgur.com/a/M0tqq), or 2) in a new settings section ("Third-party integrations"?), or 3) under global, which is kind of a sink of all kind of settings.